pkg/podinformer/podinformer.go

```diff
@@ -212,21 +213,34 @@ func (p *PodInformer) syncToStdout(key string) error {
 			if s.ContainerID == "" {
 				continue
 			}
+
+			var containerID string
+			containerIDSplit := strings.Split(s.ContainerID, "://")
+			if len(containerIDSplit) == 1 {
+				containerID = containerIDSplit[0]
+			} else if len(containerIDSplit) == 2 {
+				containerID = containerIDSplit[1]
+			} else {
+				log.WithFields(log.Fields{
+					"containerid": containerID,
```

Comment:
`containerID` is still "" at this point.

```suggestion
					"containerid": s.ContainerID,
```

Reply:
Done.